apps/download-service/src/app/modules/regulation-documents/regulation-documents.controller.ts

```diff
@@ -47,45 +49,43 @@ export class RegulationDocumentsController {
     @Body() resource: GetRegulationDraftDocumentDto,
     @Res() res: Response,
   ) {
-    let regulation
+    let draftRegulation: DB_RegulationDraft | null = null
     try {
-      regulation = await this.regulationsAdminApiService.getDraftRegulation(
+      draftRegulation = await this.regulationsAdminApiService.getDraftRegulation(
         regulationId,
         `Bearer ${resource.__accessToken}`,
       )
     } catch (e) {
       console.error('unable to get draft regulation', e)
     }
 
-    if (!regulation) {
+    if (!draftRegulation) {
       return res.status(500).end('Unable to generate pdf')
     }
 
     const input: RegulationPdfInput = {
-      title: regulation.title,
-      text: regulation.text,
+      title: draftRegulation.title,
+      text: draftRegulation.text,
       appendixes: [], // TODO where do we get these?
```

Comment:
@osk, I believe `regulationsAdminApiService.getDraftRegulation` now returns a full-featured RegulationDraft with appendixes and all the shiny bits resolved.